src/components/DayPickerRangeController.jsx

```diff
@@ -747,6 +752,23 @@ export default class DayPickerRangeController extends React.Component {
     });
   }
 
+  getNavigationStatus(visibleMonth) {
```

Comment:
Hmm, I'm not sure that this name is clear. I'd almost rather have two methods, one that asks if we should disable prev month navigation and one that asks about next month navigation.

Reply:
I think it should be kept as one method, as the only difference between them would be the arguments `(date, visibleMonth)`. Maybe `shouldDisableMonthNavigation` ?